app/views/propertyLinking/chooseEvidence.scala.html

```diff
@@ -33,11 +35,16 @@
     hasErrors = form.hasErrors
 ) {
 
+    @if(form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummary(errorList = form.errors.asTextErrorLinks, title = Text(messages("error.propertyRepresentation.title"))))
+    }
+
+    @assessmentHeading(address, localAuthorityRef)
+
+    <h2 id="caption-add-property" class="govuk-caption-l" >@messages("caption.add.property")</h2>
+
                 @formWithCSRF(action = controllers.propertyLinking.routes.ChooseEvidenceController.submit) {
 
-                    @if(form.errors.nonEmpty) {
-                        @govukErrorSummary(ErrorSummary(errorList = form.errors.asTextErrorLinks, title = Text(messages("error.propertyRepresentation.title"))))
-                    }
                     @govukRadios(Radios(
```

Comment:
Can you add html `id`s to the radio buttons, please

Reply:
Thanks for comments. I will work on it